CC/Sounder/include/constants.h

```diff
@@ -66,59 +79,4 @@ class Consts {
       -1, -1, 1,  -1, -1, -1, -1, -1, -1, -1, 1,  -1, -1, 1,  1,  -1,
       -1, -1, -1, 1,  -1, 1,  1,  1,  1,  -1, 1,  1,  -1, -1, 1,  1,
       -1, 1,  -1, -1, 1,  -1, -1, 1,  -1, -1, 1,  1,  -1, -1, -1};
-  /*
```

Comment:
Should we place these somewhere?  Are we calculating them instead?  Anyhow if we do, they should be static const

Reply:
Yes, we are calculating these from frequency-domain values. So need to keep these any more.